packages/sdks/output-generation/angular-dynamic-renderer-generator.js

```diff
@@ -0,0 +1,196 @@
+import fs from 'fs';
```

Comment:
Oh, you did create a generator, but you're using it to manually create the override and commit the output.

How about, instead of that, you add it to the build pipeline of the angular SDK, and make it write to `output/angular/src/components/dynamic-renderer/dynamic-renderer.ts` instead of `overrides/angular/src/components/dynamic-renderer/dynamic-renderer.ts`.

This way the override can stay simple!

Reply:
ahh best! great idea. Let me update it